src/app/components/navbar/navbar.component.html

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
@CryceTruly do we need this line ?.

Reply:
thanks, @alexxsanya, I have fixed this

> <img alt="Screenshot 2019-08-29 at 14 26 08" width="1111" src="https://user-images.githubusercontent.com/21230051/63936681-17bab680-ca69-11e9-9d97-a39357ddbbc7.png">
> 
> @CryceTruly. I have fetched the branch locally and installed the dependencies but when I run
> `npm run start` or `ng serve` I get the above error. Kindly check it out.

@alexxsanya  lets sync I see how you are doing it.

Reply:
For anyone that faces this, please do a clean install using `npm run install:clean` and all will be fine thereafter.